operators/olm_v1/olmv1-installing-an-operator-from-a-catalog.adoc

```diff
@@ -1,22 +1,27 @@
 :_content-type: ASSEMBLY
 [id="olmv1-installing-an-operator-from-a-catalog"]
-= Installing an Operator from a catalog in OLM 1.0 (Technology Preview)
+= Installing an Operator from a catalog in {olmv1} (Technology Preview)
 include::_attributes/common-attributes.adoc[]
 :context: olmv1-installing-operator
 
 toc::[]
 
 Cluster administrators can add _catalogs_, or curated collections of Operators and Kubernetes extensions, to their clusters. Operator authors publish their products to these catalogs. When you add a catalog to your cluster, you have access to the versions, patches, and over-the-air updates of the Operators and extensions that are published to the catalog.
 
-In the current Technology Preview release of Operator Lifecycle Manager (OLM) 1.0, you manage catalogs and Operators declaratively from the CLI using custom resources (CRs).
+In the current Technology Preview release of {olmv1-first}, you manage catalogs and Operators declaratively from the CLI using custom resources (CRs).
 
-:FeatureName: OLM 1.0
+:FeatureName: {olmv1}
 include::snippets/technology-preview.adoc[]
 
 [id="prerequisites_olmv1-installing-an-operator-from-a-catalog"]
 == Prerequisites
 
 * Access to an {product-title} cluster using an account with `cluster-admin` permissions
++
+[NOTE]
+====
+For {product-title} 4.14, documented procedures for {olmv1} are CLI-based only. Alternatively, administrators can create and view related objects in the web console by using normal methods, such as the *Import YAML* and *Search* pages. However, the existing *OperatorHub* and *Installed Operators* pages do not yet observe {olmv1} components.
```

Comment:
```suggestion
For {product-title} 4.14, documented procedures for {olmv1} are CLI-based only. Alternatively, administrators can create and view related objects in the web console by using normal methods, such as the *Import YAML* and *Search* pages. However, the existing *OperatorHub* and *Installed Operators* pages do not yet display {olmv1} components.
```

Reply:
I may be misconstruing the meaning of observe here, but if the intent is to say that the *OperatorHub* and *Installed Operators* pages don't show olmv1 components, I would suggest use "display" instead of "observe" just because "The pages don't observe" sounds a little anthropomorphic to me.